docs/usage_and_configuration/file_collection_and_discovery.md

```diff
@@ -22,10 +22,11 @@ run. The file is non-portable. The standard location on common operating systems
 `file-mode` is an int flag that determines whether the file was formatted as 3.6+ only,
 as .pyi, and whether string normalization was omitted.
 
-To override the location of these files on macOS or Linux, set the environment variable
-`XDG_CACHE_HOME` to your preferred location. For example, if you want to put the cache
-in the directory you're running _Black_ from, set `XDG_CACHE_HOME=.cache`. _Black_ will
-then write the above files to `.cache/black/<version>/`.
+To override the location of these files on all systems, set the environment variable
+`BLACK_CACHE_DIR` to the preferred location. Alternatively on macOS and Linux, set
+`XDG_CACHE_HOME` to you your preferred location. For example, if you want to put the
+cache in the directory you're running _Black_ from, set `BLACK_CACHE_DIR=.cache/black`.
+_Black_ will then write the above files to `.cache/black`.
```

Comment:
It may be implied in how this is structured (in which case I'm totally missing it) but it'd be nice to also note that `BLACK_CACHE_DIR` takes priority over `XDG_CACHE_HOME`.

Reply:
Agreed! Think this will be a definite improvement!

Reply:
addressed